templates/embedding/index.html

```diff
@@ -1,206 +1,221 @@
-      <ul class="p-list">
-        <li class="p-list__item is-ticked">The combination of embedder-only Support and specific tailored solution use-cases means Canonical can offer OEM-type pricing to embedders. Contact us to discuss your project</li>
-        <li class="p-list__item is-ticked">Regular shipment reporting to ensure your entire estate is under Canonical support coverage</li>
-      </ul>
+  </section>
+
+  <section class="p-section">
+    <hr class="is-fixed-width p-rule" />
+    <div class="p-section--shallow">
+      <div class="row--50-50">
+        <div class="col">
+          <h2>Embedding support</h2>
+        </div>
+        <div class="col">
+          <p>An efficient and clean support model for you and your customers</p>
+        </div>
+      </div>
     </div>
-    <div class="col-3 u-hide--small u-hide--medium u-align--center u-vertically-center">
-      {{
-        image (
-        url="https://assets.ubuntu.com/v1/d127abd7-safety+reliability.svg",
-        alt="",
-        width="150",
-        height="150",
-        hi_def=True,
-        loading="lazy"
-        ) | safe
-      }}
+    <div class="p-section--shallow">
+      <div class="row">
+        <div class="col-start-large-4">
+          <div class="p-section--shallow">
+            <div class="p-image-container">
+              {{ image(url="https://assets.ubuntu.com/v1/3ff2dd9f-embedding-support.png",
+                            alt="",
+                            width="1832",
+                            height="764",
+                            hi_def=True,
+                            loading="lazy",
+                            attrs={"class": "p-image-container__image"}) | safe
+              }}
+            </div>
+          </div>
+          <hr class="p-rule--muted" />
+          <div class="row">
+            <div class="col-4">
+              <p>The Embedder delivers support to their customers</p>
+            </div>
+            <div class="col-4">
+              <p>Canonical L3 Engineering support to the embedder’s L3 Engineering</p>
+            </div>
+            <div class="col-4">
+              <p>Essential, Standard and Advanced Support Tiers</p>
+            </div>
+          </div>
+        </div>
+      </div>
     </div>
-  </div>
-</section>
-
-<section class="p-strip--light">
-  <div class="row">
-    <div class="col-6 u-hide--small u-hide--medium u-align--center u-vertically-center">
-      {{
-        image (
-        url="https://assets.ubuntu.com/v1/c4b290c8-Contact+us.svg",
-        alt="",
-        width="211",
-        height="150",
-        hi_def=True,
-        loading="lazy"
-        ) | safe
-      }}
+    <div class="u-fixed-width">
+      <div class="p-cta-block">
+        <div class="row">
+          <div class="col-start-large-8">
+            <a href="/support" class="p-button">Get support</a>
```

Comment:
```suggestion
            <a href="/support" class="p-button" aria-label="Get support for your full open source stack">Get support</a>
```

Reply:
An aria label like this gives context to people using screen reader for the link they are clicking